### ISSUE TITLE
Mouseless needs restart to work normally when I turn on my computer.

### ISSUE THREAD

**Issue by satayyeb** [jbensmann/mouseless#51]
I should execute `systemctl restart mouseless.service` command every time I turn on my computer. Have you ever encountered this problem?

**Comment by jbensmann** [comment]
Yes I also have this problem when using an external keyboard, but not configure the devices in the mouseless config. The problem is that mouseless starts before the device for the external keyboard is there. I have not found a better way yet than waiting a second before starting mouseless, here is my service file:
```
[Unit]
Description=mouseless service
After=dev-input.target
Wants=dev-input.target

[Service]
ExecStartPre=/bin/sleep 1
ExecStart=/usr/local/bin/mouseless

[Install]
WantedBy=default.target
```

I will also add this to the README.

**Comment by UlyssesZh** [comment]
Setting `After=systemd-udev-settle.service` works for me. No need to `sleep`.

**Comment by jbensmann** [comment]
This should be no problem any more, as mouseless now detects new devices when they are created.
